### ISSUE TITLE
network weight decay regularizer of the loss function

### ISSUE THREAD

**Issue by BenedictGreen** [nuclearboy95/Deep-SVDD-Tensorflow#2]
Hi,
Thanks for your implementaion of the deep svdd. 
I have a question for implementation of the Deep-SVDD loss function in this code.

I think the "class DeepSVDD (line29-line36)" in 'dsvdd.deepSVDD' denote the two diiferent loss function for Deep-SVDD.
```
        with task('Build graph'):
            self.x = tf.placeholder(tf.float32, [None] + list(input_shape))
            self.latent_op = self.keras_model(self.x)
            self.dist_op = tf.reduce_sum(tf.square(self.latent_op - self.c), axis=-1)
            

            if self.objective == 'soft-boundary':
                self.score_op = self.dist_op - self.R ** 2
                penalty = tf.maximum(self.score_op, tf.zeros_like(self.score_op))
                self.loss_op = self.R ** 2 + (1 / self.nu) * penalty

            else:  # one-class
                self.score_op = self.dist_op
                self.loss_op = self.score_op

            opt = tf.train.AdamOptimizer(lr)
            self.train_op = opt.minimize(self.loss_op)
```
but this function's objective function looks different from the paper's equation (3) and equation (4) which is missing the network weight decay regularizer. 
Is it a right implementation for eq(3) and eq(4)? Or "the network weight decay is exist at other part of the code which
was ignored by me.

Thanks again!
Sincerely!
